src/content/docs/more-integrations/best-practices-integration.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: Best practices for integration users
+tags:
+  - Best practices
+  - Integrations
+metaDescription: Here are some best practices for controlling how you expose your New Relic data when you're using an integration.
+freshnessValidatedDate: 2024-03-25
+---
+
+As a New Relic administrator, you must operate your New Relic organization with the principle of least privilege. By following this principle, you can reduce the risk of unauthorized access to sensitive data. This includes assigning permissions based on job roles and responsibilities. In a [New Relic organization](/docs/accounts/accounts-billing/account-structure/new-relic-account-structure/), account-based access allows you to control what information is available to the users in your organization. This ensures that sensitive data remains secure and only authorized personnel can view it. 
+
+Dealing with integrations with third-party systems can complicate application access. For example, users in the other system may not have access to New Relic. The system may also lack centralized permissions or controls. Learn about best practices for data sharing on this page. Take control of how you want to expose your New Relic data during integrations.
```

Comment:
Some more description on what this "other system" is, the wording is kind of confusing.

I think you can cut the last sentence as well.